src/proto/deps/IdentityRpc.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+// protobuf vscode extension: https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3
```

Comment:
- Rename file to ClaimsIdentityRpc.proto to be more descriptive
- Move to folder: Identity